### ISSUE TITLE
feature id in attributes

### ISSUE THREAD

**Issue by apezel** [t-rex-tileserver/t-rex#110]
Is it necessary to export fid in feature's attributes ? I didn't checked the MVT specification but removing fid from attributes reduces significantly the size of tiles with many features (for example tiles with thousands of points)

**Comment by pka** [comment]
Feature id's are only included, when the `fid_field` is declared in the layer. More documentation needed...

**Comment by apezel** [comment]
exactly but my point is that it's included twice : as a feature id and as an attribute. This can be avoided by editing postgis.rs for the postgis datasource :
```rust
fn attributes(&self) -> Vec<FeatureAttr> {
        let mut attrs = Vec::new();
        for (i, col) in self.row.columns().into_iter().enumerate() {
            if col.name()
                != self
                    .layer
                    .geometry_field
                    .as_ref()
                    .unwrap_or(&"".to_string())
            && col.name()
                != self
                    .layer
                    .fid_field
                    .as_ref()
                    .unwrap_or(&"".to_string())
            {
```

**Comment by pka** [comment]
Ah, you're right, that's nonsene.

**Comment by pka** [comment]
Fixed in 3d8dff3, thanks!
